changelog.d/8793.doc

```diff
@@ -0,0 +1 @@
+Fixed the example on how to set the `Content-Type` header in nginx for Client Well-Known URI.
```

Comment:
```suggestion
Fix the example on how to set the `Content-Type` header in nginx for Client Well-Known URI.
```

Reply:
I followed the example from the contributing docs, that stated

> "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from EventStore to EventWorkerStore."

Should the template be updated to not use the past tense in the example?